src/main/java/org/osc/manager/ism/api/IsmDomainApi.java

```diff
@@ -53,23 +53,19 @@ public static IsmDomainApi create(ApplianceManagerConnectorElement mc, Transacti
     @Override
     public DomainEntity getDomain(String domainId) throws Exception {
 
-        return this.txControl.supports(new Callable<DomainEntity>() {
+        return txControl.supports(new Callable<DomainEntity>() {
```

Comment:
Looks like your eclipse settings have changed? this should a this. as it did before. You can set your eclipse to always include `this` for fields. See https://github.com/opensecuritycontroller/community/blob/master/development/eclipse.md#save-actions

Reply:
Yes I have changed in the settings in eclipse to make 'this' for method and field if needed, due to the previous remarks, as it was introducing the classname.this for example IsmDomainApi.this.em.find(DomainEntity.class, Long.parseLong(domainId)) instead of em.find(.....); This code style is similar to Device Member code.
so would like to retain this...

Reply:
OK, I have changed the eclipse settings as needed/suggested in the above link. Due to which eclipse adds this qualifier like for example ==> IsmDomainApi.this.em.find(... and update like this.

Reply:
OK I have fixed it now for the above not to prepend with classname.this and changes are as before and corrected this.txcontrol as suggested.